src/blog/2023/09/custom nodes.md

```diff
@@ -0,0 +1,121 @@
+---
```

Comment:
Can we update the file name to not have any spaces, and have more description for SEO?

`use-private-custom-nodes-with-flowfuse.md`?

Reply:
Also, the directory is now wrong, it should be the `10` directory

Reply:
Done